python-sdk/examples/opencv/opencv.py

```diff
@@ -86,13 +90,13 @@ def main():
 
 
 def clean():
-    client = Client()
+    client = Client.from_config()
```

Comment:
We should probably localize the Client creation in case someone edits the `main` Client but forgets this one. Maybe pass the client as an argument to these functions and move the client creation and that large comment to the `if __name__ == "__main__":` section? Or it could live as a global at the top of the file so its more visible